Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -295,6 +290,9 @@ - (void)viewDidAppear:(BOOL)animated
 
 - (void)updateBarButtonItems
 {
+    if (self.homeViewMode != HomeViewMode_Inbox) {
```

Comment:
You can't start a new conversation or access app settings from "archive mode" home view.